Add monochrome icon

apiote opened the pull request:
This adds a themed icon for launchers in Android >= 13
(https://developer.android.com/develop/ui/views/launch/icon_design_adaptive)

michelesalvador commented:
Thank you for your pull request!
I don't have a physical device with Android >= 13: so I tested on the emulators.
As I can see to obtain a themed icon there is no need to create a new _icon_foreground_mono.xml_ drawable.
The only needed line in _icon.xml_ is:
```
<monochrome android:drawable="@drawable/icon_foreground" />
```
So reusing the already existing _icon_foreground.xml_.

Do you agree?

apiote commented:
Indeed. I checked when there's no additional drawable and the monochrome line uses the existing `icon_foreground` and it works at my end.
I updated the PR — deleted the unnecessary drawable and changed `icon.xml`

michelesalvador commented:
Perfect. Thank you!
